check-module-version/action.yml

```diff
@@ -0,0 +1,51 @@
+name: 'Check module version'
+description: >
+  Сhecks the equality of the version from Lua 
+  module's code and the last repository tag
+
+inputs:
+  module-name:
+    description: 'Lua module name as when using require() in tarantool'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Clone the module
+      uses: actions/checkout@v3
+
+    - name: Setup tarantool
+      uses: tarantool/setup-tarantool@v2
+      with:
+        tarantool-version: '2.10'
+
+    - name: Make rock
+      run: tarantoolctl rocks make
+      shell: bash
+
+    - name: Code version
+      run: |
+        MODULE_VERSION=$(tarantool -e "print(require('${{ inputs.module-name }}')._VERSION)")
+        echo Module name is ${{ inputs.module-name }}
+        echo Detected version from code is $MODULE_VERSION
+        if [ "$MODULE_VERSION" = "nil" ]; then
+          echo "Check the version value implementation in the Lua module's code"
```

Comment:
It doesn't look like an error message.

Reply:
Done, now using `::error::` and `::notice::`

Reply:
I meant different. Your message describes rather some kind of stage than error. Shouldn't it be something like this:

```
The version value in the Lua module's code is undefined
```

Reply:
Updated